kubernetes-deployment/massdriver.yaml

```diff
@@ -123,14 +181,19 @@ ui:
   ui:order:
     - namespace
     - image
-    - resource_requests
-    - autoscaling
+    - resourceRequests
+    - replicas
     - port
     - ingress
-  autoscaling:
+  resourceRequests:
+    memory:
+      ui:field: conversionFieldData
+      default: Gibibytes
```

Comment:
what should the default memory be? ~100MB 🤷 

People usually think in whole numbers, then unit.  I think we should a sane-default to this field and have this widget match that default unit. 

```
100 
0.1
```

Reply:
I'm actually planning on leaving both this and CPU blank without defaults. There's really no way for us to even guess what default memory and CPU should be. We require both fields, so users will have to input it.

If we default it, we certainly may make it "easier" to just get up and running, but if we go too high we're wasting resources and if we go too low there's going to me massive resource contention on the nodes. This is very specific to the application they are running.